src/swarm/neo/node/Connection.d

```diff
@@ -23,7 +23,7 @@ class Connection: ConnectionBase
     import swarm.neo.node.RequestSet;
     import swarm.neo.protocol.connect.NodeConnect;
     import swarm.neo.authentication.HmacDef: Key;
-    import swarm.neo.connection.YieldedRequestOnConns;
+    public import swarm.neo.connection.YieldedRequestOnConns;
```

Comment:
Did you copy this decision (to use `public import`) from somewhere else (e.g. ocean)? It"s something we've traditionally avoided, but I may have missed that it's now being used in certain circumstances.

Reply:
This was not copied from any other project. I did not know that we are avoiding this.

Would you prefer to not use `public imports` here?

Reply:
As far as I remember, in other library locations where we came across this, we added the imports in the locations where they're actually used.